### ISSUE TITLE
`gulp clang-format` fails on Windows

### ISSUE THREAD

**Issue by jelbourn** [angular/angular#1415]
Low priority, but running `clang-format` (implicitly via `serve.js.dev`) on Windows complains and fails:

```
Error: FATAL: This module doesn't bundle the clang-format executable for your platform. (win32_x64)
Consider installing it with your native package manager instead.
```

It would be nice if it handled this more gracefully.

**Comment by mprobst** [comment]
@jelbourn do you have access to a Windows machine?

There are docs on how to build Clang (including `clang-format`) here: http://clang.llvm.org/get_started.html

**Comment by Bogatinov** [comment]
My environment:
* [ClangFormat 3.7](http://llvm.org/builds/) installed (added to PATH)
* Patched `gulp-clang-format` with TS support
* Added as pre-commit hook to git

**Comment by Bogatinov** [comment]
My environment:
* Windows 8.1 Pro x64
* Visual Studio code /w TypeScript 1.5beta
* Latest Node 0.12.4

How I configured:
1. Installed [Python 2.7](https://www.python.org/downloads/release/python-2710/)
2. Installed [ClangFormat 3.7](http://llvm.org/builds/) + Add to PATH
3. Patched `gulp-clang-format` in `Installation_Path\LLVM\bin` to support TypeScript
4. Add as [pre-commit hook](https://github.com/angular/angular/blob/master/DEVELOPER.md#formatting)

Every file that `clang-format` formats with `.clang-format`, it makes the [Travis CI build fail](https://travis-ci.org/angular/angular/jobs/64089876). The `gulp-clang-format` hook changes not properly formatted files and locally `gulp check-format` says is OK, while Travis CI fails for each of those files.

There are no errors when I run `clang-format` manually neither does the hook. It finds the `.clang-format` file, the TS is patched so I have reason to believe that this is Windows specific since Travis is UNIX-based machine and there are no issues with other UNIX developers.

**Comment by mprobst** [comment]
@Bogatinov I think you're just running with an outdated `clang-format` that doesn't handle arrow functions well, I'll see that I update the NPM package with the latest.

**Comment by Bogatinov** [comment]
@mprobst in [formatting](https://github.com/angular/angular/blob/master/DEVELOPER.md#formatting) it states 
```
To install, first patch this file to add .ts to the default_extensions section.
```
so I added `.ts` to the list for support. I have the latest 3.7 version from their website and running with that.

**Comment by alexeagle** [comment]
You should exclusively be using the clang-format which is distributed
through node, and referenced from the package-shrinkwrap.json file, to
avoid version skew problems.

**Comment by mlaval** [comment]
I used to have issues too when running `gulp check-format`.
But they have been solved since the release 1.0.15 of clang-format, where the Windows binary was aligned with the other ones.
A fresh `npm install` should solve the issue.

On a side note, I had to install the **Visual C++ Redistributable Packages for Visual Studio 2013** on my machine, the 32 bits version (vcredist_x86.exe). Otherwise, MSVCP120.dll was missing.
